packages/api-client/src/helpers/sendRequest.ts

```diff
@@ -103,18 +103,14 @@ export async function sendRequest(
     data: request.body,
   }
 
-  const axiosRequestConfig: AxiosRequestConfig = proxyUrl
-    ? {
-        method: 'POST',
-        url: proxyUrl,
-        data: requestConfig,
-      }
-    : {
-        method: requestConfig.method,
-        url: requestConfig.url,
-        headers: requestConfig.headers,
-        data: requestConfig.data,
-      }
+  const axiosRequestConfig: AxiosRequestConfig = {
```

Comment:
@hanspagel I think we need to handle the response stuff here ✨ and the check if using old proxy